acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -48,5 +48,15 @@
 install_packages_on(master, MASTER_PACKAGES)
 install_packages_on(agents, AGENT_PACKAGES)
 
+agents.each do |agent|
+  if agent['platform'] =~ /windows/
+    arch = agent[:ruby_arch] || 'x86'
+    base_url = "http://builds.puppetlabs.lan/puppet/#{ENV['SHA']}/artifacts/windows"
+    filename = "puppet-agent-#{ENV['VERSION']}-#{arch}.msi"
```

Comment:
I thought about that, but this change would be in puppet/master for 4.0 where we know the MSI will have `-x86` (as opposed to being able to acceptance test earlier versions of puppet)